### PR TITLE
Activity Log - Updated formatting and styles of the server credentials form

### PR DESCRIPTION
**Pull request by MichaelArestad**
#### Changes
* Rearranged the way the FormLabel was wrapping the components to match how it was used elsewhere in settings.
* Removed the table layout
* Added `div`s for the two rows of fields and styles to give them appropriate widths at appropriate sizes
* Adjusted spacing of buttons
* Added spacing to private key textarea

#### To test
1. Go to calypso Security settings for a Jetpack site
2. In the top `Backups and restores` group, Click the big plus
3. Fill out the form
4. Try to break the form
5. Resize the window a bit

#### Before
image
image

#### After
image
image

### CODE REVIEW

File: client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx [hunk 1]
```diff
@@ -78,158 +78,136 @@ export class CredentialsForm extends Component {
...
+					>
+						<option value="ssh">{ translate( 'SSH' ) }</option>
+						<option value="sftp">{ translate( 'SFTP' ) }</option>
+						<option value="ftp">{ translate( 'FTP' ) }</option>
+					</FormSelect>
+				</FormFieldset>
+
+				<div className="credentials-form__row">
+					<FormFieldset className="credentials-form__server-address">
+						<FormLabel hrmlFor="host-address">{ translate( 'Server Address' ) }</FormLabel>
```
Comment (eliorivero) [inline client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx]: Typo, should be `htmlFor`

File: client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx [hunk 2]
```diff
@@ -78,158 +78,136 @@ export class CredentialsForm extends Component {
...
+						<div>
+							<FormTextArea
+								name="kpri"
+								value={ get( this.state.form, 'kpri', '' ) }
+								onChange={ this.handleFieldChange }
+								disabled={ formIsSubmitting }
+								className="credentials-form__private-key"
+							/>
+							<p className="form-setting-explanation">
+								This field is only required if your host uses key based authentication.
```
Comment (eliorivero) [inline client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx]: Isn't this string supposed to be translatable?

**Comment by MichaelArestad** [comment]
Thanks for the review @eliorivero! Fixing all that stuff!

File: client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx (hunk 2 above)
Comment (MichaelArestad) [reply client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx]: Yes. Fixing.

File: client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx (hunk 1 above)
Comment (MichaelArestad) [reply client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx]: Fixing!

**Comment by MichaelArestad** [comment]
@eliorivero added a bit of styling to make it look better on smaller screens, fixed the typo, and attempted translation. Let me know if I did that last part right.

**Comment by apeatling** [comment]
@MichaelArestad Could you merge this one? Thanks.

**Pull request merged**